sqlmesh/core/context_diff.py

```diff
@@ -42,26 +42,26 @@ class ContextDiff(PydanticModel):
     """Whether the currently stored environment record is in unfinalized state."""
     create_from: str
     """The name of the environment the target environment will be created from if new."""
-    added: t.Set[str]
+    added: t.Set[SnapshotId]
```

Comment:
maybe this refactor would have been simplier if you just used the fqn as the str instead of changing it to snapshot id